scipy/diff/numdifftools/_step_generators.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import division
+import numpy as np
+EPS = np.finfo(float).eps
+
+
+def _generate_step(
```

Comment:
Then you can simply re-break the lines right? No need to start with an empty line after the bracket